core/tx_pool.go

```diff
@@ -621,9 +621,8 @@ func (pool *TxPool) validateTx(tx *types.Transaction, local bool) error {
 	if err != nil {
 		return ErrInvalidSender
 	}
-	// Drop non-local transactions under our own minimal accepted gas price or tip.
-	pendingBaseFee := pool.priced.urgent.baseFee
-	if !local && tx.EffectiveGasTipIntCmp(pool.gasPrice, pendingBaseFee) < 0 {
+	// Drop non-local transactions under our own minimal accepted gas price or tip
+	if !local && tx.GasTipCapIntCmp(pool.gasPrice) < 0 {
```

Comment:
I think it should be `GasFeeCapIntCmp` here.

Reply:
Actually, not sure yet.

Reply:
After some more discusssion, it seems just reverting to `GasTipCapIntCmp` is the best choice for now.

Reply:
WHY？I think tx.EffectiveGasTipIntCmp(pool.gasPrice, pendingBaseFee) < 0 Is better here.